.circleci/canary-version.js .js

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
```

Comment:
This file has an extra ` .js` in its name.

Also be sure you `chmod +x` it so that `- run: .circleci/canary-version.js` works.